src/Sulu/Bundle/WebsiteBundle/Controller/WebsiteController.php

```diff
@@ -72,18 +72,22 @@ protected function renderStructure(
 
             // if not preview enable cache handling
             if (!$preview) {
-                // mark the response as either public or private
-                $response->setPublic();
-
-                // set the private and shared max age
-                $response->setMaxAge(240);
-                $response->setSharedMaxAge(960);
-
-                // set reverse-proxy TTL (Symfony HttpCache, Varnish, ...)
-                $response->headers->set(
-                    HttpCache::HEADER_REVERSE_PROXY_TTL,
-                    $response->getAge() + intval($structure->getCacheLifeTime())
-                );
+                if (!$this->getRequest()->isMethod('GET')) {
+                    $response->setPrivate();
```

Comment:
Yeah, this PR is irrelevant now I have added this check to the new event subscriber: https://github.com/sulu-cmf/sulu/pull/653/files#diff-a601e69b47c9329ab040d289dfea324fR100